### DISCUSSION TITLE
In channel search

### DISCUSSION THREAD

**Discussion by R3tr0R4z0r** [element-hq/element-meta#2576] [Ideas]
It would be nice to implement an in chat/room feature to be able to search back messages.

The feature is implemented in fluffychat however element is a better and stable platform which would benefit the app.

**Comment by oldu9412** [comment]
Yes, it's very important, it's a blocking issue for moving to Element X. Many people won't want to come to Matrix/Element and will prefer to go to Discord, because the search function is very important to them.

**Comment by andrewbauman** [comment]
I'd have a thousand users on telegram wanting to move to elementX but search is blocking this migration till it works.
